model-optimizer/extensions/front/tf/CTCLossReplacement.py

```diff
@@ -78,53 +59,23 @@ def pattern(self):
             ])
 
     def replace_sub_graph(self, graph: Graph, match: dict):
-        seq_len_tf = match['seq_len']
         transpose_tf = match['transpose']
         ctc_greedy_decoder_tf = match['ctc_greedy_decoder']
         cast_tf = match['cast']
         ctc_loss_tf = match['ctc_loss']
         sparse_to_dense_tf = match['sparse_to_dense']
+        ctc_data_permute = create_op_with_const_inputs(graph, Transpose, {1: int64_array([1, 0, 2])},
```

Comment:
The `Transpose` operation should be added depending on the value of `logits_time_major` attribute of CTCLoss operation. It may happen that the data is already in correct layout. Refer to the documentation https://www.tensorflow.org/api_docs/python/tf/nn/ctc_loss